### DISCUSSION TITLE
contributors metadata not accessible in edit mode

### DISCUSSION THREAD

**Discussion by h-mayorquin** [dandi/helpdesk#39] [Q&A]
Hi, so yesterday (2021-11-08) I succesfully added funding information from a variety of sources to a [repository](https://dandiarchive.org/dandiset/000166/draft) This information was added in metadata -> contributors -> organization.  The results are displayed correctly on the main page
![fuding_correct](https://user-images.githubusercontent.com/6429509/140975374-2c4798f4-a634-41fd-a092-08f4add687cb.png)

However, when I try to access the edition tab in contributors to modify some of the data today (2021/11/09) I find that the information is not display and also can't not be access for edition:

![not_displaying_properly](https://user-images.githubusercontent.com/6429509/140975789-f0561f05-c0d5-4c87-836d-a7f3aad649f5.png)

The same problem is found in the subject matter of the dataset. Information is properly display in the main page but inability to edit or view in the metadata edition page.  

Has anyone encounter this problem before?

**Comment by satra** [comment]
@h-mayorquin - filed an issue (https://github.com/dandi/dandiarchive/issues/962) - this looks like a bug.

**Reply by satra** [reply]
could you please check that you still cannot load the contributor metadata?

**Reply by h-mayorquin** [reply]
Now I can load them, I am not sure what happen.

**Reply by satra** [reply]
we don't know for sure, but glad it all works now :)
